forge/db/models/Device.js

```diff
@@ -603,6 +603,16 @@ module.exports = {
                             await device.save()
                         })
                     }
+                },
+                countByState: async (states, teamId) => {
```

Comment:
You are passing the hashId from the api through - this will need converting to the database id